File: src/main/java/com/desarrollo/dto/tarea/TareaResponseDTO.java
package com.desarrollo.dto.tarea;

import lombok.Builder;
import lombok.Data;
import lombok.Getter;
import lombok.Setter;

import java.util.Date;

@Data
@Builder
@Getter
@Setter

public class TareaResponseDTO {
    private Long id;
    private String nombre;
    private Date fechaVencimiento;
    private String prioridad;
    private String proyectoNombre;
    //estado
}

File: src/app/modelos/TareaResponseDTO.ts
/**  private Long id;
    private String nombre;
    private Date fechaVencimiento;
    private String prioridad;
    private String proyectoNombre;
    private String resultado;
    
    private List<ComentarioResponseDTO> comentarios;
    private List<String> usuarios;
    */

import { ComentarioResponseDTO } from "./ComentarioResponseDTO"

export interface TareaResponseDTO{
  
    id : number,
    nombre : string,
    fechaVencimiento : Date
    prioridad  :string,
    proyectoNombre : string,
    estado : string
    comentarios : ComentarioResponseDTO[],
    usuarios : string[]


}